tests/dummy/app/templates/demos/sl-menu.hbs

```diff
@@ -194,7 +188,7 @@ export default Ember.View.extend({
 
         // need to capture TAB key with keydown event
         Ember.$( document ).on( 'keydown.menu', function( e ) {
-            if ( 9 === e.keyCode && this.get( 'controller.keyboardInUse' ) ) {
+            if ( 9 === e.keyCode &amp;&amp; this.get( 'controller.keyboardInUse' ) ) {
                 e.preventDefault();
 
```

Comment:
Why don't we just link to the source file in question, instead of having to keep this code snippet up to date?